pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/defaultvalues/SingularDefaultValues.java

```diff
@@ -26,14 +27,14 @@ public class SingularDefaultValues extends BasicDefaultDataMaker {
 
     @Override
     protected boolean setDefaultData() {
+        // TODO: this seems wrong but Singular is at end of life so did not want to mess with things - DC
         if (isParticipantDataInES()) {
-            return false;
+            throw new ESMissingParticipantData("Participant does not yet have profile and activities in ES");
         }
         return insertDefaultEnrollmentStatusForParticipant();
     }
 
     private boolean isParticipantDataInES() {
-        logger.info("Participant does not have profile and activities in ES yet...");
         return elasticSearchParticipantDto.getProfile().isEmpty() && elasticSearchParticipantDto.getActivities().isEmpty();
```

Comment:
Singular default values never made it into production, maybe we can get rid of these classes in general?

Reply:
If we are sure this is obsolete I can definitely remove the Singular handling for this message.

Reply:
Yeah, Singular is dead, and if you look at `makeDefaultable` it does not have a case for singular. We added the code and then they changed their mind

Reply:
I removed SingularDefaultValues in a commit yesterday. As you said, it was unused and just missed that in the first commit.